components/tag-list/test/tag-list.visual-diff.js

```diff
@@ -73,6 +79,17 @@ describe('d2l-tag-list', () => {
 					await page.waitForTimeout(500);
 				});
 
+				beforeEach(async() => {
+					await page.$eval(selector, async(elem) => {
```

Comment:
The impact of moving this is that "Show Less" becomes focused in the test `is correct when show more button clicked if applicable`, which seems to be the expected behaviour (based on how it behaves in the `filter-tags` tests). I'm not totally sure why it wouldn't have been focusing before.